internal/controller/controllers/agent_controller_test.go

```diff
@@ -1451,10 +1451,11 @@ VU1eS0RiS/Lz6HwRs2mATNY5FrpZOgdM3cI=
 		mockClientFactory     *MockSpokeK8sClientFactory
 	)
 	newAciWithUserManagedNetworkingNoSNO := func(name, namespace string) *hiveext.AgentClusterInstall {
+		userManagedNetworking := true
 		return &hiveext.AgentClusterInstall{
 			Spec: hiveext.AgentClusterInstallSpec{
 				Networking: hiveext.Networking{
-					UserManagedNetworking: true,
+					UserManagedNetworking: &userManagedNetworking,
```

Comment:
if you are using it only once you can do 
```suggestion
					UserManagedNetworking: swag.Bool(true),
```